test/services/CategorisationServiceSpec.scala

```diff
@@ -231,30 +231,6 @@ class CategorisationServiceSpec extends SpecBase with BeforeAndAfterEach {
 
   "calculateResult" - {
```

Comment:
I think this is my last nitpick :D 

Additional test cases to consider
-> line of code not covered

    "return Category 1" - if Nirms Assessment + Cat 1 answerable question and you answer No to a Category 1 question


-> these were the ones that were going off into odd cases before. We already have the case that its Nirms + lots of other stuff but the Nirms on its own case is a separate code path
    
    "return Category 2" - if only Nirms Assessment + Trader does not have Nirms
    "return Standard" - if Only Nirms Assessment and Trader does have Nirms

Reply:
More tests added